libs/application/ui-shell/src/lib/ApplicationForm.tsx

```diff
@@ -35,7 +40,10 @@ const ShellWrapper: FC<{
         const template = await getApplicationTemplateByTypeId(
           application.typeId,
         )
-        if (template !== null) {
+        if (
```

Comment:
Will this just show a blank page ? We could also redirect to a 404 page ?

Reply:
yes this will show a blank page and this is how it is currently. We have an asana task for rendering better error states and loading states and what not